tests/config/identifier-registration.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+# OpenID Connect client registry.
+clients:
+  - id: phoenix
+    name: OCIS
+    application_type: web
+    insecure: yes
+    trusted: yes
+    redirect_uris:
+      - http://phoenix:9100/oidc-callback.html
+      - http://phoenix:9100/
+    origins:
+      -  http://phoenix:9100
```

Comment:
extra space?
we also should be able to use `https://phoenix:9200` here see https://github.com/owncloud/phoenix/pull/3453

Reply:
I think for that we have to run ocis-proxy but, which if we run the main ocis binary runs automatically but here we have to wireup the services individually so it not running. It is somewhat of an minimal setup.